packages/fether-react/src/App/App.js

```diff
@@ -79,28 +133,36 @@ class App extends Component {
       <ReactResizeDetector handleHeight onResize={this.handleResize}>
         <div className='content'>
           <div className='window'>
-            <Router>
-              <Switch>
-                {/* The next line is the homepage */}
-                <Redirect exact from='/' to='/accounts' />
-                <Route path='/accounts' component={Accounts} />
-                <Route path='/onboarding' component={Onboarding} />
-                <Route path='/tokens/:accountAddress' component={Tokens} />
-                <Route
-                  path='/whitelist/:accountAddress'
-                  component={Whitelist}
-                />
-                <Route
-                  path='/backup/:accountAddress'
-                  component={BackupAccount}
-                />
-                <Route
-                  path='/send/:tokenAddress/from/:accountAddress'
-                  component={Send}
-                />
-                <Redirect from='*' to='/' />
-              </Switch>
-            </Router>
+            <Modal
+              title='New version available'
+              description={`${this.state.newRelease.name &&
```

Comment:
just `this.state.newRelease && this.state.newRelease.name`

Reply:
if newRelease.name is `undefined`, then it'll display "undefined was released!".
perhaps we should change this to the following, which doesn't show the description if the release doesn't have a name, but the user still knows there's a new version available since it still displays "New version available" that prompts them to click the link and view the releases page:
```
description={newRelease && newRelease.name
  ? `${newRelease && newRelease.name} was released!`
  : ''
}
```

Reply:
We have `visible={newRelease && !newRelease.ignore}` just after so we're assured that if the modal shows up, then newRelease is true-ish (and if newRelease is true-ish then newRelease.name is defined) ; made a change all the same!